Features3/tests/features/addContact.feature

```diff
@@ -0,0 +1,35 @@
+Feature: Add new contact
+    As a user
+    I want to add contact 
+    So that it will be displayed in contact list
+
+    Background:
+        Given User is logged in and navigated to Contact Fox page
+
+    Scenario Outline: Add contact with full details
+        When User enters name as "<name>", email as "<email>", phone as "<phone>" and contact type as "<contact-type>"
+        Then contact with name as "<name>", email as "<email>", phone as "<phone>" and contact type as "<contact-type>" should be created and displayed in contact list
+        Examples:
+            |name|email        |phone     |contact-type|
+            |abc |abc@gmail.com|9801234567|personal    |
+            |xyz |xyz@gmail.com|9874561230|professional|
+
+    Scenario Outline: Add contact without full details
+        When User enters name as "<name>", email as "<email>", phone as "<phone>" and contact type as "<contact-type>"
+        Then contact with name as "<name>", email as "<email>", phone as "<phone>" and contact type as "<contact-type>" should not be created 
```

Comment:
i think with email and name we can create a contact (no need of phone number) pleas check once manaully